test/integration/simple_example/controls/forseti.rb

```diff
@@ -0,0 +1,156 @@
+
+control 'forseti-client-vm' do
+  impact 1.0
+  title 'Test forseti client vm'
+  describe google_compute_instance(
+    project: project_id,
+    zone: 'us-central1-c',
+    name: forseti_client_vm_name
+  ) do
+    it { should exist }
+    its('machine_size') { should eq 'n1-standard-2' }
+  end
+end
+
+
+control 'forseti-server-vm' do
+  impact 1.0
+  title 'Test forseti server vm'
+  describe google_compute_instance(
+    project: project_id,
+    zone: 'us-central1-c',
+    name: forseti_server_vm_name
+  ) do
+    it { should exist }
+    its('machine_size') { should eq 'n1-standard-2' }
+  end
+end
+
+control 'forseti-cloudsql-instance' do
+  impact 1.0
+  title 'Test CloudSQL Instance is up'
+  describe google_sql_database_instances(project: project_id) do
+    its('instance_names') { should include /forseti-server-db-*/ }
+  end
+end
+
+
+control 'forseti-server-iam-roles' do
+
+  impact 1.0
+  title 'Test Server Project IAM Role bindings'
+  describe google_project_iam_binding(project: project_id, role: "roles/storage.objectViewer") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/storage.objectCreator") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/cloudsql.client") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/cloudtrace.agent") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/logging.logWriter") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/iam.serviceAccountTokenCreator") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+end
+
+# @TODO Need to figure out the Org level permissions to execute this test
+#
+# control 'forseti-server-read-iam-roles' do
+#   gcp_enable_privileged_resources = '1'
+#
+#   impact 1.0
+#   title 'Test Server Read Org Level IAM Role bindings'
+#   describe google_project_iam_binding(project: , role: "roles/compute.securityAdmin") do
+#     it { should exist }
+#     its ('members'){ should include /forseti/ }
+#   end
+# end
+
+control 'forseti-google-storage-buckets' do
+  impact 1.0
+  title 'Test GCS Buckets are present'
+  describe google_storage_buckets(project: project_id) do
+    its('bucket_names') { should include forseti_server_gcs_bucket }
+    its('bucket_names') { should include forseti_client_gcs_bucket }
+    its('bucket_names') { should include /forseti-cai-export/ }
+  end
+
+  describe google_storage_bucket_objects(bucket: forseti_server_gcs_bucket) do
+    let(:files) do
+      template_dir = File.expand_path(
```

Comment:
`files` should be a static list. Generating this list dynamically means that removing files from the rules directory will not cause a failure, which isn't desirable if those files have all been intentionally included.

Reply:
Addressed in 9586f46a2c159b71aa8b3d9aa0db7e8349a3cbac.